### ISSUE TITLE
How can I use this Font on Figma

### ISSUE THREAD

**Issue by JennyLan** [primefaces/primeicons#506]
Hi, I've been trying to us this font on Figma.
I can load other Icon fonts, but can't seem to get primevue to work.
Is it supported on Figma and how can I use it?

Thanks

**Comment by JennyLan** [comment]
Yes, I can get the SVGs to work in Figma.

I'm trying to replace all SVG instances using the primevue font because the padding around the actual icons are different from when it's using a font. (See attached screenshots)
In this example, to get the actual icon size to match the design, our dev team had to use 12 size font and add extra padding.

The above screenshot is the instance when I try to use the primevue font, it haven't been working for me after installing it.

![Screen Shot 2021-10-21 at 3 48 50 PM](https://user-images.githubusercontent.com/11919353/138347040-f4a95441-3bc1-4bc3-854a-72c8ae16b5f0.png)
![Screen Shot 2021-10-21 at 3 48 44 PM](https://user-images.githubusercontent.com/11919353/138347044-97cbc70c-5c62-4bb3-a3db-d00606683a9a.png)

**Comment by onursenture** [comment]
> > > I've been using the raw-svg files, but I noticed that the padding around the icon in the svg files are different from the icons when inserted as a font, and it's been causing confusion in our team what size the icon should actually be.
> > 
> > 
> > It's natural. Icon fonts and raw-svgs' doesn't share the same height/weight. You should choose one of them to ensure consistency. I recommend you to stick with raw-svg folder.
> 
> Are the icons in the raw-svgs the same font size?

No, they don't. I understand your problem. I'll discuss with my colleagues and get back to you asap. Thanks.

**Comment by onursenture** [comment]
Just to be clear, 24x24 is the svg, and 11.29x6.39 is the font-icon. Right?
